jest/config/web.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
   displayName: 'web',
   moduleNameMapper: {
     '\\.svg$': '<rootDir>/jest/mock/svg.js',
+    '\\.css$': '<rootDir>/jest/mock/file.js',
```

Comment:
As we a need a [direct css import](https://github.com/bl00mber/react-phone-input-2#usage) i've added this module.